MMETestHost/ViewController.m

```diff
@@ -0,0 +1,32 @@
+//
+//  ViewController.m
+//  MapboxMobileEvents
+//
+//  Created by Dane Miluski on 6/10/20.
+//  Copyright © 2020 Mapbox. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
```

Comment:
I experienced some difficulty debugging in the App delegate so Included this ViewController class as a place to experiment with NSUserDefaults. Although it could be removed from this PR, it may prove useful in the future to experiment with data sharing potential.

Reply:
nice @dmiluski I agree on the experiment. You can never too be cautious with NSUserDefaults.